fixtures/cameo/root-par-6.json

```diff
@@ -0,0 +1,789 @@
+{
+  "$schema": "https://raw.githubusercontent.com/OpenLightingProject/open-fixture-library/master/schemas/fixture.json",
+  "name": "ROOT PAR 6",
+  "shortName": "CLROOTPAR6",
+  "categories": ["Color Changer", "Dimmer", "Strobe"],
+  "meta": {
+    "authors": ["Goetz", "JonahKr"],
```

Comment:
@JonahKr Is this fine for you? Would you like to appear with another name instead?

Reply:
Thank you so much @FloEdelmann and @JonahKr!
From my side, I cannot contribute further and all your changes are fine with me!

Best regards
Goetz

Reply:
This is fine for me 👍